packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -121,23 +124,19 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
   /**
    * @notice Updates an oracle value and the median.
    * @param token The address of the token for which the Celo Gold exchange rate is being reported.
-   * @param numerator The amount of tokens equal to `denominator` Celo Gold.
-   * @param denominator The amount of Celo Gold equal to `numerator` tokens.
+   * @param value The amount of tokens equal to Celo Gold represented as a FixidityLib fraction
```

Comment:
It this comment accurate?  It seems `value` is also expected to have been multiplied by `getDenominator()`.  Is this what we want?  Now that we're using Fixidity it seems we could keep the internal denominator private and unexposed?

Reply:
param description fixed, internal denominator is private, but I have added a getter needed by the cli and contractskit